ui/packages/consul-ui/app/routes/dc/acls/index.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+import Route from 'consul-ui/routing/route';
+
+export default class DcAclsIndexRoute extends Route {
+  beforeModel() {
+    this.replaceWith('dc.acls.tokens');
```

Comment:
since you are creating a brand new route it would make sense not to introduce new deprecations so we have to inject router service instead of using route methods on the route it self so you would have something like:


```
@service router;

beforeModel() {
    this.router.replaceWith('dc.acls.tokens');